### PR TITLE
TASK: Use class constants and clean up code style

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

**Comment by kdambekalns** [comment]
Something for @kitsunet and @daniellienert maybe?

File: Classes/TYPO3/Party/Domain/Model/AbstractParty.php [hunk 1]
```diff
@@ -24,52 +23,55 @@
...
-	public function __construct() {
-		$this->accounts = new ArrayCollection();
-	}
+abstract class AbstractParty
+{
+    /**
+     * A unidirectional OneToMany association (done with ManyToMany and a unique constraint) to accounts. This is
+     * required to not have any dependencies from Account to AbstractParty (the other way round).
+     *
+     * @var Collection<\TYPO3\Flow\Security\Account>
```
Comment (daniellienert) [inline Classes/TYPO3/Party/Domain/Model/AbstractParty.php:32]: Does something speak against using the relative namespaces also in the collection / array syntax?

File: Classes/TYPO3/Party/Domain/Model/AbstractParty.php [hunk 2]
```diff
@@ -24,52 +23,55 @@
...
-	 * Assigns the given account to this party.
-	 *
-	 * @param Account $account The account
-	 * @return void
-	 */
-	public function addAccount(Account $account) {
-		$this->accounts->add($account);
-	}
+    /**
+     * Constructor
```
Comment (daniellienert) [inline Classes/TYPO3/Party/Domain/Model/AbstractParty.php:39]: Shouldn't we avoid comments, that do not add any information at all?

Comment (kdambekalns) [reply Classes/TYPO3/Party/Domain/Model/AbstractParty.php:39]: Sure, but the "reformat code" tool of PhpStorm doesn't detect that. :)

File: Classes/TYPO3/Party/Domain/Model/AbstractParty.php (hunk 1 above)
Comment (kdambekalns) [reply Classes/TYPO3/Party/Domain/Model/AbstractParty.php:32]: AFAIK not. Just wasn't done semi-automatically…

**Comment by bwaidelich** [comment]
Looks good (didn't test yet). Thanks for taking care

**Pull request merged**